ligo/README.md

```diff
@@ -3,3 +3,10 @@ SPDX-FileCopyrightText: 2020 tqtezos
 SPDX-License-Identifier: MIT
 -->
 # stablecoin LIGO implementation
+
+Implementation of a token smart contract in LIGO according to the [specification](/docs/specification.md).
+
+## Build Instructions
+
+1. Make sure you have `ligo` executable in your `$PATH` or set `LIGO` environment variable to the command you use to invoke `ligo`.
+2. Run `make`, it should produce `stablecoin.tz`.
```

Comment:
I think it's better to clarify its exact path

Reply:
But `stablecoin.tz` is the exact path, isn't it? If you run `make`, it will produce `stablecoin.tz` in the current folder.

Reply:
Ah, I thought that we agreed to move generated contract to `haskell/test/resources` as was done in one of our PRs, ok then.

Reply:
`make` in `ligo` produces `stablecoin.tz` (in current directory). If you call `make test` from the root or from `haskell/`, it will build `stablecoin.tz` and then symlink it to `haskell/test/resources`.